src/main/java/software/aws/neptune/gremlin/GremlinConnectionProperties.java

```diff
@@ -1120,4 +1134,32 @@ protected void validateProperties() throws SQLException {
     public boolean isSupportedProperty(final String name) {
         return SUPPORTED_PROPERTIES_LIST.contains(name);
     }
+
+    /**
+     * Updates the SERVICE_REGION env variable in JVM to the region set by Gremlin connection properties
+     */
+    @SuppressWarnings({ "unchecked" })
+    private static void setRegionEnv(final String region) throws Exception {
+        if (region.equals(System.getenv("SERVICE_REGION"))) {
+            return;
+        }
+        if (System.getenv("SERVICE_REGION") != null) {
+            LOGGER.warn(String.format("Overriding the current SERVICE_REGION environment variable with %s.", region));
```

Comment:
Why warning? As it is a standard flow should it be info level?

Reply:
Right, I was using this for easy debug, will change it.